karma.conf.js

```diff
@@ -6,7 +6,79 @@
  * the test files one-off for just a single run. This is appropriate for a
  * CI environment or if you're not otherwise running `npm run dev|hot`.
  */
-var webpackCfg = require("./webpack.config.test");
```

Comment:
GLOBAL: Please try and undo all this copy and paste stuff and refactor to reuse webpack configs and share across karma like it was before. Thanks!